lib/web/plug/session.ex

```diff
@@ -21,8 +21,9 @@ defmodule Antikythera.Plug.Session do
   alias Antikythera.Conn
   alias Antikythera.Session
 
-  defun load(conn :: v[Conn.t()], opts :: Keyword.t(String.t() | atom)) :: Conn.t() do
-    key = opts[:key]
+  defun load(conn :: v[Conn.t()], opts :: Keyword.t(String.t() | atom | map)) :: Conn.t() do
```

Comment:
> Type signature of `load/2` 's `opts` variable.

I think we can define types of each option and use them instead of `Keyword.t`.

```elixir
  @type load_option_t ::
          {:key, String.t()} | {:store, atom} | {:set_cookie, Http.SetCookie.options_t()}

  defun load(conn :: v[Conn.t()], opts :: v[list(load_option_t)]) :: Conn.t() do
```

Reply:
Addressed this in e534b4c